src/test/regress/expected/multi_utilities.out

```diff
@@ -66,3 +66,115 @@ SELECT master_apply_delete_command('DELETE FROM sharded_table');
 
 -- drop table
 DROP TABLE sharded_table;
+-- VACUUM tests
+-- create a table with a single shard (for convenience)
+CREATE TABLE dustbunnies (id integer, name text);
+SELECT master_create_distributed_table('dustbunnies', 'id', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('dustbunnies', 1, 2);
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+-- add some data to the distributed table
+\copy dustbunnies from stdin with csv
+-- run VACUUM and ANALYZE against the table on the master
+VACUUM dustbunnies;
+ANALYZE dustbunnies;
+-- verify that the VACUUM and ANALYZE ran
+\c - - - :worker_1_port
+SELECT pg_sleep(.500);
```

Comment:
I spent _so_ long looking for some clear way to do this, but I was looking in the code base, not the tests. Kind of annoying there's no API for this.

I'll copy-paste and use within my tests. Thanks for the pointer!